src/components/App.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import MenuBar from './MenuBar.jsx' ;
+import PlayGrid from './PlayGrid.jsx' ;
+
+const ROWS = 10;
+const COLS = 10;
+
+
+function create2DArray(ROWS, COLS){
+	var rows=[];
+	
+	for(var i=0; i<ROWS; i++){
+		rows.push([]); 
+	
+		for(var j=0; j<COLS; j++){
+			rows[i].push(undefined);
+		}
+	}
+	return rows;
+}
+
+
+const allLetters = "aaaaaaaaaaaaabbbcccddddddeeeeeeeeeeeeeeeeeefffgggghhhiiiiiiiiiiiijjkklllllmmmnnnnnnnnoooooooooopppqqrrrrrrrrrsssssstttttttttuuuuuuvvvwwwxxyyyzz";
```

Comment:
Single-quoted strings are generally preferred in JS. This way you have the flexibility to write HTML/JSON inside your strings.